samanthas_telegram_bot/api_queries/auxil/data_maker.py

```diff
@@ -0,0 +1,111 @@
+import json
+
+from telegram import Update
+
+from samanthas_telegram_bot.api_queries.auxil.constants import DataDict
+from samanthas_telegram_bot.data_structures.context_types import UserData
+
+
+class DataMaker:
+    """Class for creating data dictionaries to be sent to backend."""
+
+    STATUS_AT_CREATION_STUDENT_TEACHER = "awaiting_offer"
+    STATUS_AT_CREATION_TEACHER_UNDER_18 = "active"
+
+    @staticmethod
+    def personal_info(user_data: UserData) -> DataDict:
```

Comment:
Imho, these methods should be part of `UserData` class, why you decided to put them to external class? 

Current approach is still not bad, we can keep it as it is probably. But I think it's better to call it DataConverter or just Converter, because it's not `making` DataDict from `nothing`, but converts UserData to DataDict.

You can skip this if you want, because this naming problems are not so important, just from my point of view this name sounds strange

Reply:
Hey, this is a great idea. Done